src/components/ProfileDisplay.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import CommunityGrid from './CommunityGrid';
+import StudentGrid from './StudentGrid';
+
+export default function ProfileDisplay() {
+  const [user, setUser] = React.useState(0);
+
+  const handleChange = (event, newUser) => {
+    setUser(newUser);
+  };
+
+  function displaySelection(userSelected) {
+    if (userSelected === 0) {
+      return <StudentGrid />
+    }
+    return <CommunityGrid />
+  }
+
+
+  return (
+    <div className="stepperStyle">
+      <Paper square>
```

Comment:
The paper is not necessary. What you should do instead is set the `backgroundColor` of the surrounding div to `theme.palette.background.paper`

Reply:
`<div className="stepperStyle" backgroundColor= theme.palette.background.paper >` This didnt work ill leave it pending but I did remove the `<Paper square>`